src/types/routeMethods.spec-d.ts

```diff
@@ -407,25 +407,6 @@ test('public parent routes have correct type for parameters', () => {
   }>()
 })
 
-test('param names must be alphanumeric', () => {
```

Comment:
Another thing I noticed is the `UnifyParamEnds` type reduced the number of parameters that could be parsed. I don't think the use case here is strong enough to accept the performance cost and limiting the number of params. So changing my mind on this "feature" and voting to remove and make `/` the only end character.

Reply:
Is this the same type that we use to parse params in the query though? If so we will need "&" as a possible end character as well.